atom/browser/api/atom_api_web_contents.h

```diff
@@ -183,6 +183,7 @@ class WebContents : public mate::TrackableObject<WebContents>,
   void SetFrameRate(int frame_rate);
   int GetFrameRate() const;
   void Invalidate();
+  gfx::Size GetSizeForNewRenderView(content::WebContents*) const override;
```

Comment:
This seems to be called form the chromium side, but I'm not sure why this is here, maybe @gerhardberger can help me out.